airbyte-integrations/connectors/source-facebook-marketing/integration_tests/test_streams.py

```diff
@@ -32,16 +31,22 @@ def state_with_include_deleted_fixture(state):
 
 
 @pytest.fixture(scope="session", name="configured_catalog")
-def configured_catalog_fixture():
-    return ConfiguredAirbyteCatalog.parse_file("integration_tests/configured_catalog_without_insights.json")
+def configured_catalog_fixture(config) -> ConfiguredAirbyteCatalog:
+    catalog = SourceFacebookMarketing().discover(logger=logging.getLogger("airbyte"), config=config)
+    streams = [
+        ConfiguredAirbyteStream(stream=stream, sync_mode=stream.supported_sync_modes[0], destination_sync_mode=DestinationSyncMode.append)
```

Comment:
why are we always using `supported_sync_modes[0]`? is the idea that we use incremental if available otherwise full_refresh? if so why not do something like 

```
# Prefer incremental if available
sync_mode="incremental" if "incremental" in stream.supported_sync_modes else "full_refresh"
```

Reply:
no, the idea is to use explicit sync mode, but I see that in some places we just leave it as is

Reply:
updated